### DISCUSSION TITLE
Forced to use Next Image component instead of native html img tag

### DISCUSSION THREAD

**Discussion by joginayak** [vercel/next.js#26792] [Help]
Hi,

I am trying to deploy a NextJS project on Vercel and got this error:

**Error: Do not use <img>. Use Image from 'next/image' instead. See https://nextjs.org/docs/messages/no-img-element.**

Does this mean I am now forced to use the Image component? I have my own lazy loading and image optimisation solutions and prefer to use them instead. Why am I being forced to use the Image component?

Is there any work around for this? Please let me know. 

Thanks!

**Comment by Dragate** [comment]
You're not forced to use the Image component. This error is coming from Next.js custom eslint rules, which you can disable. You can see the rules [here](https://nextjs.org/docs/basic-features/eslint#eslint-plugin) and how to disable specific ones [here](https://nextjs.org/docs/basic-features/eslint#disabling-rules).

**Reply by joginayak** [reply]
Understood. Thanks @Dragate.
